src/mpi/controller.rs

```diff
@@ -109,6 +110,17 @@ pub fn run(world: SystemCommunicator, config: Config) {
     let duration = end.sub(start).as_millis() / 1000;
     info!("#{rank} took: {duration}s");
 
+    info!("output dir: {:?}", config.output_dir);
+
+    if config.routing_mode == RoutingMode::AdHoc {
+        match remove_dir_all(config.output_dir + "routing") {
```

Comment:
I think this is a case for `expect`. Or do you really want to proceed if the tmp folder can't be removed?

Reply:
fixed it in #48